examples/xonotic/gameserver.yaml

```diff
@@ -23,5 +23,5 @@ spec:
     spec:
       containers:
       - name: xonotic
-        image: gcr.io/agones-images/xonotic-example:0.1
+        image: gcr.io/agones-images/xonotic-example:0.2
         # imagePullPolicy: Always # add for development
```

Comment:
Note part of the diff technically, but since you're in the neighborhood, maybe add a newline here?

Reply:
Done